src/applications/edu-benefits/0993/containers/SubmitButtons.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import moment from 'moment';
+import ProgressButton from 'us-forms-system/lib/js/components/ProgressButton';
+import { timeFromNow } from 'us-forms-system/lib/js/utilities/date';
+
+export default function SubmitButtons(props) {
+  const {
+    onBack,
+    onSubmit,
+    submission,
+    renderErrorMessage
+  } = props;
+  let submitButton;
+  let submitMessage;
+
+  if (submission.status === false) {
+    submitButton = (
+      <ProgressButton
+        submitButton
+        onButtonClick={onSubmit}
+        buttonText="Submit Application"
+        buttonClass="usa-button-primary"/>
+    );
+  } else if (submission.status === 'submitPending') {
+    submitButton = (
+      <ProgressButton
+        onButtonClick={onSubmit}
+        buttonText="Sending..."
+        disabled
+        buttonClass="usa-button-disabled"/>
+    );
+  } else if (submission.status === 'applicationSubmitted') {
+    submitButton = (
+      <ProgressButton
+        onButtonClick={onSubmit}
+        buttonText="Submitted"
+        disabled
+        buttonClass="form-button-green"
+        beforeText="&#10003;"/>
+    );
+  } else if (submission.status === 'clientError') {
+    submitButton = (
+      <ProgressButton
+        onButtonClick={onSubmit}
+        buttonText="Submit Application"
+        buttonClass="usa-button-primary"/>
+    );
+    submitMessage = (
+      <div className="usa-alert usa-alert-error schemaform-failure-alert">
+        <div className="usa-alert-body">
+          <p className="schemaform-warning-header"><strong>We’re sorry, there was an error connecting to Vets.gov.</strong></p>
+          <p>Please check your Internet connection and try again. If the problem persists, please contact the Vets.gov Help Desk.</p>
+        </div>
+      </div>
+    );
+  } else if (submission.status === 'throttledError') {
+    submitButton = (
+      <ProgressButton
+        onButtonClick={onSubmit}
+        buttonText="Submit Application"
+        buttonClass="usa-button-primary"/>
+    );
+    submitMessage = (
+      <div className="usa-alert usa-alert-error schemaform-failure-alert">
+        <div className="usa-alert-body">
+          <p className="schemaform-warning-header"><strong>We’ve run into a problem</strong></p>
+          <p>We’re sorry. Your submission didn’t go through because we received too many requests from you. Please wait {timeFromNow(moment.unix(submission.extra))} and submit your request again.</p>
+        </div>
+      </div>
+    );
+  } else {
```

Comment:
This component is the same as the us-forms-system SubmitButton except for the removal of an else statement for handling validation messaging on the review page (where errors may be hidden within a collapsed accordion), which is not necessary for this single page form as it does not include a review page:
```
  } else if (submission.status === 'validationError') {
    submitButton = (
      <ProgressButton
        onButtonClick={onSubmit}
        buttonText="Submit Application"
        buttonClass="usa-button-primary"/>
    );
    submitMessage = (
      <div className="usa-alert usa-alert-error schemaform-failure-alert">
        <div className="usa-alert-body">
          <p className="schemaform-warning-header"><strong>We’re sorry. Some information in your application is missing or not valid.</strong></p>
          <p>Please check each section of your application to make sure you’ve filled out all the information that is required.</p>
        </div>
      </div>
    );
```